packages/components/src/components/hds/tabs/tab.hbs

```diff
@@ -5,7 +5,7 @@
 {{! template-lint-disable require-context-role no-invalid-role }}
 <li class={{this.classNames}} ...attributes role="presentation">
   <button
-    class="hds-tabs__tab-button hds-typography-body-200 hds-font-weight-medium"
+    class="hds-tabs__tab-button hds-font-weight-medium"
```

Comment:
The font-size changes depending on whether `@size` is set to "medium" (the default) or "large" so I moved this to Tabs.scss

Reply:
At this point I would suggest to move also the `font-weight` in CSS, so all the text styling is in the same place.